book-cw/part-2-coding/regex.adoc

```diff
@@ -1,3 +1,348 @@
 = Coding in Regex
```

Comment:
How about a more task-based title here, like "Using regular expressions to automate your work"?

Reply:
My chapter titles are 2-3-word format. I wonder if I should reconsider that.